source/i18n/i18n.cpp

```diff
@@ -225,6 +237,12 @@ const std::string& i18n::location(Language lang, u16 v, u8 originGame)
 {
     switch (originGame)
     {
+        case 1:
+        case 2:
+        case 3:
+        case 4:
+        case 5:
```

Comment:
Just want to make absolutely certain that these are the proper values

Reply:
Going by [games.txt](https://github.com/Universal-Team/PKSM-Core/blob/gen3/strings/eng/games.txt), I think that's right